src/trufont/objects/factories.py

```diff
@@ -76,13 +76,16 @@ def pathGraphicsPathFactory(path):
     points = path._points
     if not points:
         return graphicsPath
+    if all(point.type is None for point in points):
```

Comment:
This is a hot code path. Why make a linear time assertion for a possibility that we don't know is possible?

Reply:
Hm. Guess it can be turned into a `assert not stack` after the point loop?